v1/BetterRTX_Installer.ps1

```diff
@@ -294,10 +296,37 @@ Switch ($selection) {
         $version = $releases[$SelectVersion - 1]
         $newStubUrl = $version.stub
         $newToneMappingUrl = $version.tonemapping
+
+        #   TODO: Host a separate CDN for older tonemappings.
+        #         Currently, the default tonemapper is used for all variations of BetterRTX
+
+        $oldToneMappingUrl = "https://cdn.discordapp.com/attachments/1111388664468086836/1148375947129663528/RTXPostFX.Tonemapping.material.bin?ex=663c9db1&is=663b4c31&hm=afb0a9f03211f6ce9ed004207410a2543b32437be06d8c9dbc1b12b95f49ebe1&"
```

Comment:
TODO: Setup on bedrock.graphics instead of through the discord content delivery network. I'd also prefer to be able to update on the api side without having to push a new update to the script.

Reply:
We also need to update the variable names to make them clearer.

Reply:
Umm, any suggestions? I thought “oldTonemappingUrl” was descriptive enough.

Reply:
I was moreso speaking in general across the whole script.